.gitignore

```diff
@@ -1,9 +1,10 @@
-target/
-.project
-.classpath
-.settings/
-*.ipr
-*.iml
-/tests/Test*/
-/extras/facade/Prevalence/
-.byke
+target/
+.project
+.classpath
+.settings/
+*.ipr
+*.iml
+/tests/Test*/
+/extras/facade/Prevalence/
+.byke
+.idea/
```

Comment:
Thanks for fixing the line endings in this file. :)

I thought I had fixed up line endings recently, but I now realize that I only searched in source code files and thereby missed this one.

Reply:
hehe - fixed by accident. Just added a new line `.idea/` and the diff was larger than expected.